task4/src/routes/group/index.js

```diff
@@ -0,0 +1,99 @@
+import express from 'express';
+import GroupService from '../../services/groupService';
+import db from '../../models';
+
+const router = express.Router();
+const groupService = new GroupService(db);
+
+router
+    .route('/:id')
+    .get(async (req, res) => {
+        try {
+            const { id: groupId } = req.params;
+            const group = await groupService.getById(groupId);
+            if (!group) {
+                res.status(404).json({
+                    message: 'No group found'
+                });
+            } else {
+                res.json(group);
+            }
+        } catch (err) {
+            res.status(err.status || 500).json({
+                message: err
+            });
+        }
+    })
+    .delete(async (req, res) => {
+        try {
+            const { id: groupId } = req.params;
+            const group = await groupService.getById(groupId);
```

Comment:
No need to search for a group & then delete, you can just delete & catch error

Reply:
As discussed with @meteorra, it's OK to receive success response for deleting  not existed item, so I'm getting rid of this code